server/src/main/java/io/druid/server/initialization/jetty/JettyServerModule.java

```diff
@@ -153,8 +123,9 @@ public JacksonJsonProvider getJacksonJsonProvider(@Json ObjectMapper objectMappe
     return provider;
   }
 
-  private static Server makeJettyServer(@Self DruidNode node, ServerConfig config)
+  static Server makeJettyServer(DruidNode node, ServerConfig config)
   {
+    log.info("Creating Jetty server listening at port [%s] with config [%s]", node.getPort(), config);
```

Comment:
On second thought, I can see the `ServerConfig` and port info being logged by `JsonConfigurator` and jetty server respectively so I can remove the line altogether